starter-code/index.html

```diff
@@ -26,10 +26,10 @@ <h1>Pizza Builder</h1>
           <button class="btn btn-green-peppers active">Green peppers</button>
         </li>
         <li>
-          <button class="btn btn-sauce active">White sauce</button>
+          <button class="btn btn-sauce">White sauce</button>
```

Comment:
You were not supposed to change the `.html` file
We expected you to do this using jQuery, that's cheating :stuck_out_tongue: